Assets/AnkrSDK/Runtime/WalletConnect2/RpcRequests/SilentSigning/SilentSigningConnectionRequestData.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using Newtonsoft.Json;
+using WalletConnectSharp.Common.Utils;
+using WalletConnectSharp.Network.Models;
+
+namespace AnkrSDK.WalletConnect2.RpcRequests.SilentSigning
+{
+	[RpcMethod("wallet_requestSilentSign")]
+	[RpcRequestOptions(Clock.SIX_HOURS, false, 3001)]
+	[RpcResponseOptions(Clock.SIX_HOURS, false, 3002)]
+	public class SilentSigningConnectionRequestData : RpcRequestListDataBase
+	{
+		[Serializable]
+		public class SilentSigningConnectionRequestParams
```

Comment:
Why is this class nested?

Reply:
Because we only need it inside this class, it can also be made private not only nested imo